code/game/controllers/planet.py

```diff
@@ -1,11 +1,17 @@
+from code.game.exceptions import DamageLimitException
+
+
 class PlanetController:
     MAX_HEALTH = 100
 
     def __init__(self) -> None:
         self._health = self.MAX_HEALTH
 
     def get_damage(self, damage: int) -> None:
-        self._health = max(self._health - damage, 0)
+        if self._health - damage <= 0:
+            raise DamageLimitException
+
+        self._health -= damage
```

Comment:
предлагаю проверять сразу:
```suggestion
        self._health = max(self._health - damage, 0)

        if self._health == 0:
            raise DamageLimitException
```